### PR TITLE
Improve training validations and dismiss accepted training

### PR DESCRIPTION
**Pull request by choosen**
1) Improve training validations
2) Add owner as player to training on save
3) 'dismiss accepted training' in controllers and views
4) Clean and spec controler actions permisions

### CODE REVIEW

File: app/views/trainings/_users_training_undone.html.haml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+%tr
+  %td= users_training.user.display_name
+  %td.text-success= formatted_ut_accepted_at(users_training)
+  %td.text-danger= 'x' unless users_training.attended?
+  %td
+    - if can?(:update, users_training) && !users_training.training.finished?
+      .btn-group
+        -# = link_to attended ? 'Set as absence' : 'Set as present',
```
Comment (weirdobeaver) [inline app/views/trainings/_users_training_undone.html.haml]: tu sie cos wykomentowalo

File: spec/models/ability_spec.rb [hunk 2]
```diff
@@ -16,12 +18,15 @@
       let(:user) { create(:admin) }
 
       it { is_expected.to be_able_to(:manage, :all) }
+      it do
+        # binding.pry
```
Comment (weirdobeaver) [inline spec/models/ability_spec.rb]: mozna wywalic

File: app/views/trainings/_users_training_undone.html.haml (hunk 1 above)
Comment (choosen) [inline app/views/trainings/_users_training_undone.html.haml]: to jest własnie # todo

File: spec/models/ability_spec.rb (hunk 2 above)
Comment (choosen) [inline spec/models/ability_spec.rb]: :+1:

File: app/views/trainings/show.html.haml [hunk 3]
```diff
@@ -4,7 +4,7 @@
     %h3 Basics
     = render @training
     %br
-    - if can? :update, @training
+    - if can?(:update, @training) # && @training.finished?
```
Comment (weirdobeaver) [inline app/views/trainings/show.html.haml]: comentarz do wywalenia

**Pull request merged**